plugin-server/src/cdp/workers/cyclotron.worker.ts

```diff
@@ -0,0 +1,143 @@
+import { CyclotronJob, CyclotronWorker } from '@posthog/cyclotron'
+
+import { runInstrumentedFunction } from '../../main/utils'
+import { status } from '../../utils/status'
+import { CdpConsumerBase } from '../consumers/cdp-base.consumer'
+import { counterJobsProcessed, gaugeBatchUtilization } from '../metrics/metrics'
+import { HogFunctionInvocation, HogFunctionInvocationResult, HogFunctionTypeType } from '../types'
+import { cyclotronJobToInvocation, invocationToCyclotronJobUpdate } from '../utils'
+
+/**
+ * The future of the CDP consumer. This will be the main consumer that will handle all hog jobs from Cyclotron
+ */
+export class CdpCyclotronWorker extends CdpConsumerBase {
```

Comment:
Generally with this PR I don't like that you've chosen some values like "worker" or "manager" to be special. They aren't 😅 

To me the only separation that makes sense is `consumer` (CdpCyclotronWorker is also a consumer) as this indicates its a top level service (like a deployment)

Everything else is just an internal service so I think we should just have `services/` for everything currently called a manager or an executor.

But in the end the only thing I feel strongly about is that CdpCyclotronWorker is a consumer 😅